src/tests/ftest/io/ior_small.yaml

```diff
@@ -11,7 +11,7 @@ hosts:
     - client-F
     - client-G
     - client-H
-timeout: 1800
+timeout: 3000
```

Comment:
Shouldn't the timeout be lower than 1800 if we're avoiding server start and disk formatting for each test variant ?

Reply:
3000 seconds seems to be a long time. Is it not?

Reply:
This I kept high for my initial runs of CI to make sure test finishes in CI. SO was just to be on safe side. It actually took considerably less than what I thought it would. So changing it to 1800 now as in one of CI runs it finished in 20 mints.
Also previously the timeouts were for each test variants. Now all different test variants are being run as part of single test. So having higher timeout value. Will see how long these tests consistently and we can change these values in future.